dstep/translator/MacroDefinition.d

```diff
@@ -2053,6 +2053,15 @@ bool translateFunctAlias(
     {
         Identifier ident = cast(Identifier) expr.expr;
 
+        if (ident.spelling == "assert")
```

Comment:
Yeah I checked how `assert.h` header looks on few systems and it looks like euristic that is doomed to fail. Unless libclang provides some super-magical helpers to do that I think assuming that any `assert` no matter where it comes from acts like D built-in `assert` is reasonable.